_scripts/prs.py

```diff
@@ -3,6 +3,7 @@
 Requires PyGithub.
 """
 import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
convention would be to put this after `import github`, to stress that it is not part of the std lib

Reply:
I recommend to just run a formatter (`black` or similar) over the script.

Reply:
Either sounds good. I haven't used formatters before.

Reply:
could add a pre-commit config to do it automatically

Reply:
I ran it through black. Let me know if there's anything else.